pysilcam/silcamgui/silcamgui.py

```diff
@@ -822,9 +822,9 @@ def stop_record(self):
 
         def load_stats_filename(self):
             self.stats_filename = QFileDialog.getOpenFileName(self,
-                    caption = 'Load a *-STATS.csv file',
+                    caption = 'Load a *-STATS.h5 file',
```

Comment:
E128: continuation line under-indented for visual indent  <!-- Automatically posted by Lintly -->